src/backend/gl/src/info.rs

```diff
@@ -256,41 +256,46 @@ pub fn get(gl: &gl::Gl) -> (Info, Capabilities, PrivateCaps) {
-                                                               Es  (3,2),
-                                                               Ext ("GL_ARB_draw_elements_base_vertex")]),
-        srgb_color_supported:              info.is_supported(&[Core(3,2),
-                                                               Ext ("GL_ARB_framebuffer_sRGB")]),
-        constant_buffer_supported:         info.is_supported(&[Core(3,1),
-                                                               Es  (3,0),
-                                                               Ext ("GL_ARB_uniform_buffer_object")]),
-        unordered_access_view_supported:   info.is_supported(&[Core(4,0)]), // TODO: extension
-        separate_blending_slots_supported: info.is_supported(&[Core(4,0),
-                                                               Es  (3,0),
-                                                               Ext ("GL_ARB_draw_buffers_blend")]),
-        copy_buffer_supported:             info.is_supported(&[Core(3,1),
-                                                               Es  (3,0),
-                                                               Ext ("GL_ARB_copy_buffer"),
-                                                               Ext ("GL_NV_copy_buffer")]),
+        instance_rate:                      info.is_supported(&[Core(3,3),
+                                                                Es  (3,0),
+                                                                Ext ("GL_ARB_instanced_arrays")]),
+        vertex_base:                        info.is_supported(&[Core(3,2),
+                                                                Es  (3,2),
+                                                                Ext ("GL_ARB_draw_elements_base_vertex")]),
+        srgb_color:                         info.is_supported(&[Core(3,2),
+                                                                Ext ("GL_ARB_framebuffer_sRGB")]),
+        constant_buffer:                    info.is_supported(&[Core(3,1),
+                                                                Es  (3,0),
+                                                                Ext ("GL_ARB_uniform_buffer_object")]),
+        unordered_access_view:              info.is_supported(&[Core(4,0)]), // TODO: extension
+        separate_blending_slots:            info.is_supported(&[Core(4,0),
+                                                                Es  (3,0),
+                                                                Ext ("GL_ARB_draw_buffers_blend")]),
+        copy_buffer:                        info.is_supported(&[Core(3,1),
+                                                                Es  (3,0),
+                                                                Ext ("GL_ARB_copy_buffer"),
+                                                                Ext ("GL_NV_copy_buffer")]),
+    };
+    let caps = Capabilities {
```

Comment:
not sure if `Capabilities` makes any sense now. Might consider just storing features and limits directly